ambry-router/src/test/java/com.github.ambry.router/PutManagerTest.java

```diff
@@ -438,46 +422,26 @@ public void testBadChannelPutFailure() throws Exception {
   }
 
   /**
-   * A bad arguments test, where the channel size is different from the size in BlobProperties.
-   */
-  @Test
-  public void testChannelSizeNotSizeInPropertiesPutFailure() throws Exception {
-    requestAndResultsList.clear();
-    int blobSize = chunkSize;
-    RequestAndResult requestAndResult = new RequestAndResult(blobSize);
-    // Change the actual content size.
-    requestAndResult.putContent = new byte[blobSize + 1];
-    requestAndResultsList.add(requestAndResult);
-    Exception expectedException = new RouterException("", RouterErrorCode.BadInputChannel);
-    submitPutsAndAssertFailure(expectedException, true, false);
-  }
-
-  /**
-   * A bad arguments test, where the blob size is very large.
+   * Test that the size in BlobProperties is ignored for puts, by attempting puts with varying values for size in
+   * BlobProperties.
    */
   @Test
-  public void testBlobTooLargePutFailure() throws Exception {
```

Comment:
Do we have a way to test the case where the operation fails b/c the number of chunks is greater than INTMAX? I understand that this might be hard to verify now since we have to wait for all of the data chunks to be filled.